app/views/templates/script/package_action.erb

```diff
@@ -113,7 +118,7 @@ handle_zypp_res_codes () {
     end
   -%>
   zypper refresh
-  zypper -n <%= action %> <%= input("package") %>
+  zypper -n <%= action %> <%= input("options") %> <%= input("package") %>
```

Comment:
I don't know zypper by heart, but for yum and apt the options go before the action, but in here the options follow the action. Is that intentional?

Reply:
For zypper, global options are before the action and action-specific options are after the action. Most commonly you want to set action specific options. Otherwise we would need two additional template parameters which would mood make any sense for apt/yum.